utils/make_readme_table.py

```diff
@@ -62,6 +62,7 @@ def display_name(self) -> str:
     ReleaseInfo.from_simple_tag("0.15.0", "March 18, 2020"),
     ReleaseInfo.from_simple_tag("0.15.1", "March 30, 2020"),
     ReleaseInfo("release_1", "1.0.0", "0.16.0", "April 30, 2020"),
+    ReleaseInfo("release_2", "1.0.1", "0.16.1", "May 19, 2020"),
```

Comment:
Will fix this on release_2_branch

Reply:
never mind, we don't actually use that version in the table. Fixing on this branch instead.